pkgs/development/compilers/llvm/16/default.nix

```diff
@@ -297,7 +308,12 @@ in let
       bintools = bintoolsNoLibc';
       extraPackages = [ ];
       extraBuildCommands = mkExtraBuildCommands0 cc;
-      nixSupport.cc-cflags = [ "-nostartfiles" ];
+      nixSupport.cc-cflags = [
+        "-nostartfiles"
+        # The compiler got stricter about this, so a number of downstream
+        # things seem to fail with it switched on.
+        "-Wno-implicit-function-declaration"
```

Comment:
Same as above. Are there any packages on Linux where you’re encountering problems that haven’t fixed yet on master and staging?